### DISCUSSION TITLE
Tree: MuliSelect mode, checkbox has all the A11Y props with storybook example but with sandbox implementation its not there.

### DISCUSSION THREAD

**Discussion by saorabhkr** [UI5/webcomponents-react#4356] [Q&A] [answered]
Hello, 
I can see Tree component with MuliSelect mode, rendered checkbox has proper A11y properties whereas when I was creating a code stand box for tree list item and custom list item those A11y properties are missing for the checkbox. Can someone please help me understand If anything else needs to be updated? 

Code Stand Box Ex: 
https://codesandbox.io/s/editor-proposals-forked-xky86d?file=/src/App.js
https://codesandbox.io/s/editor-proposals-forked-m87kro?file=/src/App.js

Taken for **List-Item** 
<img width="1411" alt="Screenshot 2023-03-16 at 11 16 19 AM" src="https://user-images.githubusercontent.com/5204425/225526811-fc0561aa-5933-446e-b8da-060a3922a0c9.png">

Taken for **Custom-list-Item** 
<img width="1475" alt="Screenshot 2023-03-16 at 11 16 34 AM" src="https://user-images.githubusercontent.com/5204425/225526774-3e6c88d3-1f08-4df5-ad82-f185723c1c48.png">

Images taken from storybook:
<img width="1401" alt="Screenshot 2023-03-16 at 11 22 19 AM" src="https://user-images.githubusercontent.com/5204425/225527051-d2bc264d-d9da-4448-999a-aeec0bae9228.png">
Link: https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-tree--docs

Am I missing while creating a sandbox?

Best Regards,

**Comment by Lukas742** [comment, marked as answer]
Hi @saorabhkr

you're using outdated versions in your codeSandbox where these enhancements weren't available yet. Since this component is developed by the ui5-webcomponents team, I don't know in which version the changes were made.

[Here's](https://codesandbox.io/p/sandbox/editor-proposals-forked-f4p4jd?file=%2Fsrc%2FApp.js) an updated codeSandbox using the latest version of both `@ui5/webcomponents` and `@ui5/webcomponents-react` and there the attribute is available.

**Reply by saorabhkr** [reply]
Thanks for the update, I'll give a try.
